glean-core/src/lib.rs

```diff
@@ -72,9 +72,7 @@ impl Glean {
 
     fn initialize_core_metrics(&mut self) -> Result<()> {
         if first_run::is_first_run(&self.data_path)? {
```

Comment:
Just an FYI: In https://github.com/mozilla-mobile/android-components/pull/2838, we ultimately got rid of this side-channel information for this and just set first_run whenever that metric doesn't exist in the database.  I think we defer that until we port that PR to glean-core, but just wanted to note it here.

Reply:
Do we have a bug for that in Glean-core?

Reply:
I couldn't find one, so just made one: https://bugzilla.mozilla.org/show_bug.cgi?id=1557366